lib/models/model.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/** A class to represent model, parent class of category and product
+ * @constructor
+ * @property schema (mongoose)
+ */
+class Model {
+  constructor(schema) {
+    this.schema = schema;
+  }
+
+  /**
+ * validating schema with jsonSchema
+ * returns validated schema or blank object
+ */
```

Comment:
Not accurate: This function returns the schema for the model as a JSON object

This comment is also not in correct JSDoc format; it should use the `@return` tag